core-bundle/src/Resources/views/Frontend/preview_toolbar_base_js.html.twig

```diff
@@ -299,5 +299,26 @@
                 });
             });
         });
+
+        // copy published link to clipboard
+        window.addEventListener('cto-toolbar-loaded', function() {
+            const copyButton = document.getElementById('copyPublishedPath');
+            copyButton.addEventListener('click', function(event) {
+                const input = document.createElement('input');
+                input.value = window.location.href.replace('{{ preview_script }}', '');
```

Comment:
This won't work for URLs like `https://www.domain.com/preview.php/page.html?filter=stuff/containing/preview.php/etc.`. I know it's super unlikely but it could happen and we must prevent this from failing :)
We should only replace the first occurence right after the host.

Reply:
@Toflar actually this should work as expected! `String.prototype.replace()` only replaces the first occurence. For replacing every occurence there is the newer `String.prototype.replaceAll()` that was introduced some months ago.

![Bildschirmfoto 2020-10-19 um 12 31 15](https://user-images.githubusercontent.com/26705426/96439372-03a2bd00-1207-11eb-8fc2-d2ba412d0519.png)

Reply:
Interesting. Good then :)